app/views/wiki/_wikis.html.erb

```diff
@@ -9,6 +10,7 @@
   <% @wikis.each do |wiki| %>
     <tr>
       <td style="width:50%"><i class="fa fa-file"></i> <a href="<%= wiki.path %>"><%= wiki.latest.title %></a></td>
+      <td><i><%=wiki.path%></i></td>
```

Comment:
@Pranay10rawat  please give space, like this
 `<td><i><%= wiki.path %></i></td>`